app/src/main/java/org/oppia/android/app/utility/ClickableAreasImage.kt

```diff
@@ -152,4 +156,31 @@ class ClickableAreasImage(
     )
     newView.setBackgroundResource(R.drawable.selected_region_background)
   }
+
+  fun highlightBox(clickableArea: ImageWithRegions.LabeledRegion) {
```

Comment:
Can this be consolidated with other logic in this utility? It seems non-ideal to duplicate the view creation logic here & elsewhere in the utility.

Reply:
I understand why it's needed--I'm not asking to remove it. Instead, I'm asking that you consolidate the implementation code by de-duplicating it with other similar code in this file (e.g. by introducing helper functions).

Reply:
Created a new helper function to de-duplicate the code of creating selectableView.